services/api-server/src/simcore_service_api_server/plugins/catalog.py

```diff
@@ -14,9 +14,7 @@
 from ..models.schemas.solvers import LATEST_VERSION, Solver, SolverKeyId, SolverPort
 from ..utils.client_base import BaseServiceClientApi, setup_client_instance
 
-## from ..utils.client_decorators import JSON, handle_errors, handle_retry
-
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
```

Comment:
Question: what is the idea behind turning the loggers to "private"?

Reply:
right :-)
This was after some conversations with @sanderegg and @matusdrobuliak66 regarding loggers.
Generally speaking i believe it is a good policy to "document intentions" by using code. 

Here there was a remark by @sanderegg that we want to enforce these loggers to be used within the module and not outside. Therefore the natural way to say that they are ``protected`` is to write  ``_logger`` instead of ``logger``.

From now on I will convert all of them into protected (if I have no objections!).